modules/mono/build_scripts/build_assemblies.py

```diff
@@ -300,6 +306,9 @@ def main():
     parser.add_argument("--godot-platform", type=str, default="")
     parser.add_argument("--mono-prefix", type=str, default="")
     parser.add_argument("--push-nupkgs-local", type=str, default="")
+    parser.add_argument(
+        "--float", type=str, default="32", choices=["default", "32", "64"], help="Floating-point precision"
```

Comment:
Since we removed `default`.

```suggestion
        "--float", type=str, default="32", choices=["32", "64"], help="Floating-point precision"
```